src/components/OurExperts/StyledOurExperts.ts

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+
+export const StyledExpertSectionWrap = styled.div`
+position:relative;
+padding:100px 0 120px;
+
+& a{
+    position:absolute;
+    top:auto;
+    right:100px;
+    bottom:120px;
+}
+`
+
+export const StyledSlideWrap = styled.div`
+width:228px;
+`
+
+export const StyledExpertWrap = styled.div`
+position:relative;
+display:flex;
+flex-direction:column;
+align-items:center;
+width:100%;
+transition:all 0.5s;
+margin-top: 40px;
```

Comment:
I think you could add a cursor:pointer here, since there is some kind of action when you hover over it

Reply:
@NSVoronova  Not agree, cause user might think, that he can click it. But he can't...